tests/util/test_networks.py

```diff
@@ -94,11 +104,42 @@ def test_running_norm_matches_dist(batch_size: int) -> None:
     assert running_norm.count == num_samples
 
 
+@pytest.mark.parametrize("batch_size", [1, 8])
+@pytest.mark.parametrize("normalization_layer", NORMALIZATION_LAYERS)
+def test_parameters_converge(
+    batch_size: int,
+    normalization_layer: Type[networks.RunningNorm],
+) -> None:
+    """Test running norm parameters approximately converge to true values."""
+    mean = th.Tensor([42])
+    var = th.Tensor([42])
```

Comment:
I'd suggest using a different value for mean and variance, to catch bugs of the form `self.running_mean = self.running_var` (unlikely to be something actually that dumb, admittedly, but I'd have a vague fear there can be more subtle failures like this)